src/osp/Active/ActiveScene.h

```diff
@@ -135,18 +133,27 @@ class ActiveScene
     }
 
     /**
-     * Calculate world transformations and draw to all render targets
+     * @brief Calculate world transformations and draw to all render targets
+     *
+     * @deprecated Move this to somewhere rendering/application related and
+     *             pass render targets as arguments or something
      */
     void draw();
 
     // TODO
     constexpr float get_time_delta_fixed() const { return 1.0f / 60.0f; }
 
-    Package& get_context_resources() { return m_context; }
+
+    constexpr PackageRegistry& get_packages() { return m_rPackages; };
+    constexpr PackageRegistry& get_packages() const { return m_rPackages; };
+
+    constexpr Package& get_context_resources() { return m_rContext; }
+    constexpr Package& get_context_resources() const { return m_rContext; }
```

Comment:
If they both return a non-const refernece to Package, why bother defining the non-const version of the function?

Reply:
This is probably a mistake